docs/first_pr/kitty_felix_stephan.md

```diff
@@ -0,0 +1,4 @@
+# Kitty
+#Stephan
```

Comment:
Can you make this more consistent? Space missing here.

Reply:
Hey, will do

Reply:
Fixed everything @ryaanahmed